coingecko/adapter.js

```diff
@@ -106,6 +107,35 @@ const globalMarketCap = (jobRunID, input, callback) => {
   Requester.request(config, customError).then(_handleResponse).catch(_handleError)
 }
 
+const mktdomParams = {
+  quote: ['quote', 'to', 'market'],
+}
+
+const globalMarketDom = (jobRunID, input, callback) => {
```

Comment:
We can consolidate this function with the other function using the same endpoint.

Example from the work I started on this, but never got time to finish: https://github.com/smartcontractkit/external-adapters-js/compare/master...btc-dominance

Reply:
Looks like we can do the same on coinpaprika as well